src/Chapter1.hs

```diff
@@ -554,7 +556,12 @@ value after "=" where the condition is true.
 Casual reminder about adding top-level type signatures for all functions :)
 -}
 
-mid x y z = error "mid: not implemented!"
+mid :: Int -> Int -> Int -> Int
+mid x y z
+  | (x < y && x > z) || (x > y && x < z) = x
+  | (y < z && y > x) || (y > z && y < x) = y
+  | (z < x && z > y) || (z > x && z < y) = z
+  | otherwise = if x == y then x else z
```

Comment:
You can combine these two cases under one otherwise case if you use `<=` and `>=`in the above cases 🙂 

```suggestion
  | otherwise = z
```

Reply:
Ohkay, makes sense